src/agent/debugger/src/target.rs

```diff
@@ -272,24 +207,159 @@ impl Target {
-                if let Err(e) = module.sym_load_module(self.process_handle) {
-                    error!("Error loading symbols: {}", e);
+        Ok(())
+    }
+
+    pub fn initial_wow64_bp(&mut self) {
+        self.saw_initial_wow64_bp = true;
+    }
+
+    fn try_resolve_all_unresolved_breakpoints(&mut self) {
+        // borrowck - take ownership from self so we call `try_resolve_unresolved_breakpoint`.
+        let mut unresolved_breakpoints = std::mem::take(&mut self.unresolved_breakpoints);
+        unresolved_breakpoints.retain(|bp| !self.try_resolve_unresolved_breakpoint(bp));
+        assert!(self.unresolved_breakpoints.is_empty());
+        self.unresolved_breakpoints = unresolved_breakpoints;
+    }
+
+    /// Try to resolve a single unresolved breakpoint, returning true if the breakpoint
+    /// was successfully resolved.
+    fn try_resolve_unresolved_breakpoint(&mut self, breakpoint: &Breakpoint) -> bool {
+        if !self.saw_initial_bp {
+            return false;
+        }
+
+        let mut resolved = false;
+        match breakpoint.extra_info() {
+            breakpoint::ExtraInfo::Rva(rva_info) => {
+                if let Some(module) = self.module_from_name_mut(rva_info.module()) {
+                    let address = module.base_address() + rva_info.rva();
+                    module.new_breakpoint(breakpoint.id(), breakpoint.kind(), address);
+                    resolved = true;
                 }
             }
+            breakpoint::ExtraInfo::UnresolvedSymbol(sym_info) => {
+                let process_handle = self.process_handle; // borrowck
+                if let Some(module) = self.module_from_name_mut(sym_info.module()) {
+                    match dbghelp::lock() {
+                        Ok(dbghelp) => {
+                            match dbghelp.sym_from_name(
+                                process_handle,
+                                module.name(),
+                                sym_info.function(),
+                            ) {
+                                Ok(sym) => {
+                                    module.new_breakpoint(
+                                        breakpoint.id(),
+                                        breakpoint.kind(),
+                                        sym.address(),
+                                    );
+                                    resolved = true;
+                                }
+                                Err(_) => {
+                                    debug!(
+                                        "unknown symbol {}!{}",
+                                        module.name().display(),
+                                        sym_info.function()
+                                    );
+                                }
+                            }
+                        }
+                        Err(e) => {
+                            error!("Can't set symbolic breakpoints: {}", e);
```

Comment:
```suggestion
                            error!("Can't set symbolic breakpoints: {:?}", e);
```
To include causes (and someday, backtraces when available).

Reply:
Thanks for the tip, I've been using nightly anyway and it is handy getting the backtraces.